care/facility/api/viewsets/bed.py

```diff
@@ -57,6 +59,28 @@ class BedViewSet(
     search_fields = ["name"]
     filterset_class = BedFilter
 
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        validated_data = serializer.validated_data
+        number_of_beds = validated_data.pop("number_of_beds", 1)
+        # Bulk creating n number of beds
+        if number_of_beds > 1 and number_of_beds <= 100:
+            objs = []
+            for i in range(1, number_of_beds + 1):
+                temp_data = dict(validated_data.copy())
+                temp_data["name"] = temp_data["name"] + f" - {i}"
+                objs.append(Bed(**temp_data))
+
+            res = Bed.objects.bulk_create(objs=objs, batch_size=number_of_beds)
+            return Response(res, status=status.HTTP_201_CREATED)
```

Comment:
Why would you need the response in this case? The bed names are predictable right?